cmd/storagenode/main.go

```diff
@@ -161,14 +167,15 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	overrides := map[string]interface{}{
-		"log.level": "info",
-	}
-	serverAddress := cmd.Flag("server.address")
-	if !serverAddress.Changed {
-		overrides[serverAddress.Name] = defaultServerAddr
+		"identity.cert-path":      setupCfg.Identity.CertPath,
```

Comment:
Why you reverted this block of code? Current version looks like this:
```
	overrides := map[string]interface{}{
		"log.level": "info",
	}
	serverAddress := cmd.Flag("server.address")
	if !serverAddress.Changed {
		overrides[serverAddress.Name] = defaultServerAddr
	}
```

Reply:
This wasn't meant to be changed, this PR has some weird merges in it. I'll fix this.